docs/algorithms/sig/rainbow.md

```diff
@@ -11,14 +11,6 @@
   - **Implementation license (SPDX-Identifier)**: CC0-1.0
 
 
```

Comment:
Not happy about the text removal. It is logical given that "copy_from_upstream" has run, so arguably OK for this PR, but we need to decide whether/how to persist this warning (or whether to remove Rainbow right away): Opinions? @dstebila ?

Reply:
Made up my mind finally: The security/test notes in the documentation files and eliminated by this PR should be retained as they are the only place some algorithm deficiencies are documented.

@thb-sb: Would you be willing to manually re-add those to the PR (i.e., eliminate the changes to the files below done by `copy_from_upstream`)? I'd otherwise volunteer to/will do this to finally merge the PR. Files affected:
docs/algorithms/kem/classic_mceliece.md
docs/algorithms/kem/hqc.md
docs/algorithms/sig/picnic.md
docs/algorithms/sig/rainbow.md
docs/algorithms/sig/sphincs.md

Reply:
Done: 9856ced96027056795c11bbc064f2cc0cb124f2b reverted the documentation.

Reply:
Thanks!